thirtyinch-rx2/src/main/java/net/grandcentrix/thirtyinch/rx2/RxTiPresenterUtils.java

```diff
@@ -66,4 +86,54 @@ public boolean isDisposed() {
         }).distinctUntilChanged();
     }
 
+    /**
+     * Returns a transformer that will delay onNext, onError and onComplete emissions unless a view
+     * become available. getView() is guaranteed to be != null during all emissions. This
+     * transformer can only be used on application's main thread.
```

Comment:
I must admit, when I wrote the `rx1` version I [copied](https://github.com/konmik/nucleus/blob/4167820fbb4061d8a6f303de3903751a8e136793/nucleus/src/main/java/nucleus/presenter/RxPresenter.java#L182) most of the docs.
Actually the main thread restriction is not true it's only recommended to make sure the view is always non null. Attach/Detach happens on the main thread. When code executes on any other thread it's not guaranteed that those actions happen before the view attach/detach state changes. So it may happen that the emission of `T` happens after the view got detached.

Reply:
I'm happy to rewrite the comment. The behaviour should be clear from the tests anyway, I hope.